app/javascript/packs/document-capture.tsx

```diff
@@ -172,7 +172,6 @@ const App = composeComponents(
     ServiceProviderContextProvider,
     {
       value: getServiceProvider(),
-      selfieCaptureEnabled: getSelfieCaptureEnabled(),
```

Comment:
I'd expect to see some corresponding changes in `app/javascript/packages/document-capture/context/service-provider.tsx` to remove references to the prop, if we're moving it off this component?

Reply:
Thanks, I forgot to do that. Removed [here](https://github.com/18F/identity-idp/pull/9580/commits/0a66ceef2078871d0c561cab13b300a971c0ebe7).